envpool/atari/atari_env.h

```diff
@@ -75,8 +77,9 @@ class AtariEnvFns {
   }
 };
 
+static uint8_t gray_scale_mapping[65536];
```

Comment:
Move this into the AtariEnv class.

Reply:
no it cannot be compiled successfully

Reply:
```
bazel-out/k8-fastbuild/bin/envpool/atari/_objs/atari_env_test/atari_env_test.pic.o:atari_env_test.cc:function atari::AtariEnv::PushStack(bool, bool): error: undefined reference to 'atari::AtariEnv::gray_scale_mapping'
bazel-out/k8-fastbuild/bin/envpool/atari/_objs/atari_env_test/atari_env_test.pic.o:atari_env_test.cc:function atari::AtariEnv::PushStack(bool, bool): error: undefined reference to 'atari::AtariEnv::gray_scale_mapping'
bazel-out/k8-fastbuild/bin/envpool/atari/_objs/atari_env_test/atari_env_test.pic.o:atari_env_test.cc:function atari::AtariEnv::InitMapping(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >): error: undefined reference to 'atari::AtariEnv::gray_scale_mapping'
```

Reply:
This link error can be solved by adding a definition outside of the class. An example can be find in https://github.com/sail-sg/envpool/blob/master/envpool/core/py_envpool.h#L104